### PR TITLE
Added two different projects

### PR DESCRIPTION
**Pull request by Shahil093**
## Checklist for projects addition 
 
- [x] My project has contribution guidelines.
- [x] My project has a Code of Conduct.
- [x] My project has a README file.
- [x] My project has a License.

---

Please note that if your project doesn't meet these requirements, your PR may be closed. Make sure to include these elements in your project before submitting the PR.

### CODE REVIEW

File: src/pages/AddYourProjectsGuide.jsx [hunk 1]
```diff
@@ -11,26 +11,26 @@ function AddYourProjectsGuide() {
...
         {
-          "link": "PROJECT_LINK",
-          "title": "PROJECT_NAME",
-          "description": "PROJECT_DESCRIPTION",
-          "tech": ["tech1", "tech2"]
+          "link": "https://github.com/Shahil093/Text-Utils",
+          "title": "Text-Utils / https://text-utils-website.web.app/ ",
+          "description": "This Is A Text-Utils App",
+          "tech": ["Reactjs", "HTM/CSS & BootStrapp"]
         }
```
Comment (priyankarpal) [inline src/pages/AddYourProjectsGuide.jsx:34]: why are these changes here? Please read contributing guidelines before creating a PR

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by priyankarpal** [comment]
the pull request appears to contain spam. In any open-source project, it's important to read contributing guidelines before creating a pull request. also, there is no issue number

**Comment by Shahil093** [comment]
Can you explain what is spam in it?

**Comment by priyankarpal** [comment]
> Can you explain what is spam in it?

for projects addition you need to add json format code but you have changed the docs file & there is no issue number
